packages/mongodb-memory-server-global-3.6/postinstall.js

```diff
@@ -1,19 +1,21 @@
 /* eslint @typescript-eslint/no-var-requires: 0 */
 
-function isModuleExists(name) {
+function doesModuleExist(name) {
```

Comment:
An interesting solution to move postinstall script to the core package 👍

In such case we can drastically simplify all `postinstall` scripts in`-global-` packages to the following form:
```js
var mongodb_version = require('./package.json').mongodb_version;
require('mongodb-memory-server-core/lib/utils/postinstall-helper.js').downloadBinaries({ version: mongodb_version });
```

There is no need 
- to require `setDefaultValue` method
- global directory can be determined in core package

`postinstall.js` better to rename to `./utils/postinstall-helper.js` for removing the misconception that `core` package has postinstall script and somebody forgets to call it )

Reply:
So we can remove all copy-pasta from postinstall scripts and put there just two lines
- which version needs to install
- call the autoinstall method from core package

Reply:
2a9aab7738e6512ce0e03135f4c8a8d217d62816 should have this applied